packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -16,8 +18,8 @@ async function getPathEnv(appRoot: string) {
 
   // Fish, bash, and zsh all support -i and -c flags.
   const shellPath = process.env.SHELL ?? "/bin/zsh";
-  const { stdout: path } = await execa(shellPath, ["-i", "-c", `cd "${appRoot}" && echo "$PATH"`]);
-  return path.trim();
+  const { stdout: path } = await execa(shellPath, ["-i", "-c", `cd "${appRoot}" && echo "${PRE_PATH_TOKEN}$PATH"`]);
+  return path.split(PRE_PATH_TOKEN)[1].trim();
```

Comment:
would it be possible for the shell to also add something after the command? For example, there are some extensions that print exit code after each command. Maybe we should have both prefix and suffix to extract the path?